libbeat/docs/processors-list.asciidoc

```diff
@@ -95,6 +95,9 @@ endif::[]
 ifndef::no_truncate_fields_processor[]
 * <<truncate-fields, `truncate_fields`>>
 endif::[]
+ifdef::include_translate_sid_processor[]
+* <<processor-translate-sid, `translate_sid`>>
+endif::[]
```

Comment:
@dedemorton We switched to docs/beats excluding what they don't support. Shall we be more consistent and use `ifndef::no_translate_sid_processor`?

Reply:
yes let's be consistent.

Reply:
Done.